crates/ruff_diagnostics/src/diagnostic.rs

```diff
@@ -25,7 +25,7 @@ pub struct Diagnostic {
     pub kind: DiagnosticKind,
     pub location: Location,
     pub end_location: Location,
-    pub fix: Option<Fix>,
+    pub fix: Option<Edit>,
```

Comment:
Is my understanding correct that you plan to replace `Option<Edit>` with `Option<Fix>` in a follow up PR (and thus, keeping the name `fix` makes sense).

Reply:
Yes!